src/baseline_bots/bots/smart_order_accepter_bot.py

```diff
@@ -143,6 +201,26 @@ def is_support_for_selected_orders(self, support_order: str) -> bool:
             return True
         return False
 
+    def is_support_for_given_orders(self, support_order, orders):
+        """Determine if selected support order for neighbour corresponds to given list of orders"""
+        order_tokens = get_order_tokens(support_order)
+        if order_tokens[2].split()[1] not in orders:
```

Comment:
add a comment explaining this

Reply:
I didn't write this function so I think it is best for somebody else to add comments and type hints. I can make an issue if you'd like.